src/main/java/io/kestra/plugin/weaviate/BatchCreate.java

```diff
@@ -83,6 +164,12 @@ public static class Output implements io.kestra.core.models.tasks.Output {
         )
         private List<String> className;
 
+        @Schema(
+            title = "The URI of stored data",
+            description = "Content of file will be the created objects data"
+        )
+        private URI uri;
+
         @Schema(
             title = "The properties of the created objects"
         )
```

Comment:
Please remove this as it shouldn't exist, it only should returns the createdCount and the URI